draft-ietf-scitt-architecture.md

```diff
@@ -974,15 +974,12 @@ All contents exchanged between actors is protected using secure authenticated ch
 
 #### Signed Statements and Their Registration
 
-The Transparency Service is trusted with the confidentiality of the Signed Statements presented for Registration.
-Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing.
-Others may just return Receipts to clients that present Singed Statements for Registration, and disclose the Append-only Log only to Auditors trusted with the confidentiality of its contents.
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for registration. The act of registration will always record in the
+logs the Signed Statement.  There is a use case where the logs are completely publicly accessible, and another where the audit requires elevated permission (this could be for privacy or for mitigation of servicing expensive operations). In either case the returned Receipts to the client will both facilitate simple evaluation of meeting the identity policy bar and act as a key to allow query during the elevated deep audit.
```

Comment:
Please note with your changes, we need to remove line : 980 below, as we are not registering Transparent Statements in the transparency service!

Reply:
remove or change to... see below?